draft-ietf-tls-exported-authenticator.md

```diff
@@ -181,7 +181,11 @@ Section 4.4.2.
 
 The certificate message contains an opaque string called
 certificate_request_context, which is extracted from the authenticator request if
-present.  If no authenticator request is provided, it is zero-length.
+present.  If no authenticator request is provided, the certificate_request_context
+MUST be either:
+
+* zero length
+* chosen to be unpredictable to the peer
```

Comment:
The reason for the latter being that it will be unlikely to collide with a value that they choose?  You should say as much.

Reply:
Since this can only be validated as a spontaneous authentication I don't think unpredictability is actually necessary.